android/src/main/java/com/bluechilli/flutteruploader/MethodCallHandlerImpl.java

```diff
@@ -112,10 +113,19 @@ private void enqueue(MethodCall call, MethodChannel.Result result) {
     for (Map<String, String> file : files) {
       items.add(FileItem.fromJson(file));
     }
-
```

Comment:
nit, keep this one

Reply:
The whitespace changes stem from the formatting tool. I am not sure if I need to change any settings there.